components/telemetry-operator/controllers/telemetry/logpipeline_controller.go

```diff
@@ -49,10 +48,11 @@ type LogPipelineReconciler struct {
 	FluentBitDaemonSet types.NamespacedName
 	restartsTotal      prometheus.Counter
 	unsupportedTotal   prometheus.Gauge
+	FluentBitUtils     *fluentbit.DaemonSetUtils
 }
 
-// NewLogPipelineReconciler returns a new LogPipelineReconciler using the given FluentBit config arguments
-func NewLogPipelineReconciler(client client.Client, scheme *runtime.Scheme, daemonSetConfig sync.FluentBitDaemonSetConfig, pipelineConfig fluentbit.PipelineConfig) *LogPipelineReconciler {
+// NewLogPipelineReconciler returns a new LogPipelineReconciler using the given DaemonSetUtils config arguments
```

Comment:
The DaemonSetUtils in the comment does not make sense. Guess that was renamed by using the IDE's rename feature?